src/agent/Cargo.toml

```diff
@@ -74,8 +74,8 @@ clap = { version = "3.0.1", features = ["derive"] }
 openssl = { version = "0.10.38", features = ["vendored"] }
 
 # Image pull/decrypt
-image-rs = { git = "https://github.com/confidential-containers/guest-components", rev = "53ddd632424432077e95d3901deb64727be0b4c1", default-features = false, features = [
-    "kata-cc-native-tls",
+image-rs = { git = "https://github.com/confidential-containers/guest-components", rev = "a4e739c", default-features = false, features = [
```

Comment:
We have https://github.com/kata-containers/kata-containers/pull/7852/files that is bumping image-rs to use the latest version and avoid the cargo.lock issue. I'm not sure if we need to land one in particular first, but it might be an idea to update image-rs to use the same version here to avoid mismatches?

Reply:
Sure.

Reply:
Fixed. I've update the version of image-rs in sync with https://github.com/kata-containers/kata-containers/pull/7852